cumulusci/tasks/bulkdata/generate_mapping.py

```diff
@@ -61,7 +65,7 @@ def _run_task(self):
 
     def _collect_objects(self):
         """Walk the global describe and identify the sObjects we need to include in a minimal operation."""
-        self.mapping_objects = []
+        self.mapping_objects = self.options["include"]
```

Comment:
We probably should validate that the specified names are in the describe result so the user doesn't just get a KeyError at https://github.com/SFDO-Tooling/CumulusCI/pull/1834/files#diff-d1647e48e48295d91b884a9874b6ad9fR95 if they made a typo.

Reply:
4349fb8